free-3.md

```diff
@@ -1,49 +1,79 @@
+ 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL would be appropriate for my users because what they lack is the translation from more convenient laptop keyboard inputs into MuseScore. The domain specificity of this language is important; this language does not have to perform the function of a general programming language, but what it should do is provide just enough functionality for users to translate their computer keyboard strokes into MuseScore dication. If the language was not domain specific, the learning curve might be unnecessarily steep for the function/goal of the language, and might involve a lot of unnecessary elements that are unrelated to the user’s goals. 
 
 ### Why you?
-_What excites you about this idea? How did you come up with it?_
 
+_What excites you about this idea?_
+
+I really enjoy arranging music, and would really enjoy a tool that makes the process more efficient! 
+
+_How did you come up with it?_
+
+I came up with it one day when I was really frustrated with the painstaking process of translating musical thoughts into sheet music via MuseScore, and I did an angry keyboard smash. Then, I realized how cool it would be if I could tell Musecore the tempo of my song, and then press the computer keyboard keys that correspond to the notes I wanted to dictate. Ideally, the composition software would also compare how long I hold the key down (lets say 700 milliseconds) to the BPM (tempo) of the tempo I determined, to notate not only the notes themselves but the duration of the notes.
 
 ### Domain
-_Describe the project's domain in five words._
+Music Composition 
 
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
+_How might the user interact with the language?_
+
+The language will provide a text input box that allows someone to type in the tempo in BPM, time signature, key, and clef. For example, the user could say that the song was in 96BPM per quarter note, in 4/4 time, in G Major, and in treble clef. Then, a new MuseScore file would be created, and prompt the user to decide how many voices/instruments were going to be involved in the piece. The user chooses one of the instruments, starts the metronome, and presses the keyboard keys representing the names of the notes he wants to notate, holding down the key presses for a long amount of time if the note was longer, and a short amount of time if the note was shorter. 
+
+_What does programming look like? Why is this the right way to interact with the problem domain?_ 
+
+I described a bit above about how programming would look like. I think this is the right way to interact with the problem domain, because I want the music composition process to be as fast and efficient as possible, even without a piano keyboard.
 
```

Comment:
I think this idea has great potential, but I have some questions over the specifics.  How would you represent the notes on your laptop?  Would the note C correspond to the letter C?  I think one possibility would be to have the home row represent some subset of the keys.  Then it would be possible to include accidentals also, which would be the computer keys above and in between the pairs of keys on the home row.